### ISSUE TITLE
Return proper web url, citation info

### ISSUE THREAD

**Issue by vidiecan** [ufal/lindat-kontext#95]
Expected: additional information from (lindat) corparch

Clicking on corpus title calls `ajax_get_corp_details` and then
`lindat_corparch:get_corpus_info`. If that returns proper information, it can be displayed in `overview.jsx` e.g., `_renderWebLink`

**Comment by kosarko** [comment]
@vidiecan web link should be fixed with #80; but someone needs to provide the xml attribute `web` in corplist.xml (for each corpora).

It's more complicated with citation info as that is ommited in the parser. We can cut and paste this
```
        ref_elm = node.find('reference')
        if ref_elm is not None:
            ans.citation_info.default_ref = translate_markup(getattr(ref_elm.find('default'),
                                                                     'text', None))
            articles = [translate_markup(getattr(x, 'text', None)) for x in ref_elm.findall('article')]
            ans.citation_info.article_ref = articles
            ans.citation_info.other_bibliography = translate_markup(
                getattr(ref_elm.find('other_bibliography'), 'text', None))
```
from default corparch; but do we hardwire the metadata into corplist?

Was the fix for #80 enough or is there more to do?

**Comment by vidiecan** [comment]
this has been adressed in a way already
